### ISSUE TITLE
Support Palestine

### ISSUE THREAD

**Issue by yousefelgoharyx** [reactjs/reactjs.org#4877]
Is it possible to put a banner supporting Palestine since it's under attack now from Isreal and people are dying 
I want to raise people's awareness about the Palestinian case

**Comment by YunsGeek** [comment]
yes , Ukraine is not better than Palastine 🇵🇸

**Comment by bgoonz** [comment]
Agreed…Idk how many other Jews would publicly support any initiative for the people of Palestine but honestly shame on those who offer only callous indifference  … you don’t have to be an antisemite in order not to support a proverbial  Zionist manifest destiny in the strip/West Bank; and you don’t need to have an issue with Israel’s existence to have a problem with the way Palestinian families are loosing their freedoms and sometimes even homes for (no reason that anyone could explain to anyone else without citing their personal religious beliefs. ) I hope they will find a peaceful solution after all the decades of systematic failures they have been subject to.

**Comment by jvatic** [comment]
With what's been happening in Palestine becoming more visible over the past 90 days, I'm wondering if there's enough support to get a PR approved to add this banner? (See also: https://github.com/reactjs/react.dev/issues/6504#issuecomment-1879504726)

**Comment by psmak4** [comment]
If you're going to put up a banner supporting Palestine you should also have one supporting Israel since they were attacked by Hamas which started this current incarnation of the conflict.
